### ISSUE TITLE
How to search for notes with spaces in them?

### ISSUE THREAD

**Issue by ioistired** [rr-/szurubooru#419]
I'm trying to search for this image
![image](https://user-images.githubusercontent.com/538336/124530655-8dfd7200-ddd2-11eb-9231-bcc34c84bb94.png)

It has a note that just captions the image. How would I search for it? I tried `note-text:"*have you seen this cat*"`, `"note-text:*have you seen this cat*"`, and `note-text:[*have you seen this cat*]` to no avail

**Comment by sgsunder** [comment]
Agreed, there should be an escape sequence for spaces for use in this context.

Partial workaround for now: `note-text:*have*you*seen*this*cat*`

**Comment by po5** [comment]
Implemented here, looks good to me: 6590608bcec7a3532578a59320281fd75ba8ec94
